### PR TITLE
Feature/render results title

### PR DESCRIPTION
**Pull request by mrSelieznov**
#230 Task

Added title for render results

### CODE REVIEW

**Review by mrSelieznov — COMMENTED** [review]
I set 'viewed' all files changed by prettier and leave empty my changes

File: src/components/Home.tsx [hunk 1]
```diff
@@ -1,13 +1,13 @@
-import { useEffect, useState } from 'react';
+import { useEffect, useState } from "react";
```
Comment (oleksandrstarov) [inline src/components/Home.tsx:1]: Use single quotes

File: src/components/RenderResults.tsx [hunk 2]
```diff
@@ -1,19 +1,20 @@
-import { useState } from 'react';
-import { MovieCard } from '../models/index';
-import CardInfo from './CardInfo';
-import SmallInfoCard from './SmallInfoCard';
-import useDeviceDetect from '../hooks/useDeviceDetect';
+import { useState } from "react";
```
Comment (oleksandrstarov) [inline src/components/RenderResults.tsx:1]: Use single quotes

**Review by oleksandrstarov — CHANGES_REQUESTED** [review]

File: src/styles/RenderResults.scss [hunk 3]
```diff
@@ -37,10 +37,11 @@
   }
   
   .view-toggler{
-    display: flex;
-    justify-content: flex-end;
-    padding-right: 30px;
-    margin-bottom: 20px;
+  display: flex;
```
Comment (oleksandrstarov) [inline src/styles/RenderResults.scss:40]: check indent for this block

File: src/components/Home.tsx [hunk 4]
```diff
@@ -7,7 +7,7 @@ import Container from './Container';
 import Slider from './Slider';
 import Title from './Title';
 
-import api from 'src/service/api';
+import api from "src/service/api";
```
Comment (oleksandrstarov) [inline src/components/Home.tsx:10]: Use single quotes

**Review by oleksandrstarov — CHANGES_REQUESTED** [review]

**Review by oleksandrstarov — APPROVED** [review]

**Review by oleksandrstarov — APPROVED** [review]

**Review by Kachyr — APPROVED** [review]

**Pull request merged**